gallery/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@sonatype/react-shared-components-gallery",
-  "version": "2.1.2",
+  "version": "2.1.3",
```

Comment:
That's a patch version.  A minor version bump would be 2.2.0

Reply:
> That's a patch version. A minor version bump would be 2.2.0

Oops I'll fix

Reply:
> That's a patch version. A minor version bump would be 2.2.0

Should be fixed by https://github.com/sonatype/sonatype-react-shared-components/pull/172/commits/c7c3784a194b3348b844eaecb20a5224b090a2c2